styles.css

```diff
@@ -1243,8 +1336,736 @@ footer {
   flex-grow: 0;
   margin: 17px 82px;
   list-style: none;
+  justify-content: center;
 }
 
 /* Contact Form section style end here */
 
 /* ------------------------------------------------------------------------------------------------------------------ */
+
+/* ------------------------------------------------------------------------------------------------------------------ */
+
+/* Mobile Media query section starts here */
+
+@media only screen and (max-width: 1180px) {
```

Comment:
- [ ] [OPTIONAL] Kindy make sure your design is mobile-first, currently, it's desktop first, you are supposed to use a media query to build your desktop version.